app/graphql/mutations/comment_survey.rb

```diff
@@ -18,6 +18,7 @@ def resolve(survey_id:, message:)
 
       begin
         comment.save
+        NotificationMailer.survey_commented(comment, survey).deliver_now
```

Comment:
besser devlier_later? um den Versand über den Background Job abzuwickeln? 
Andernfalls wartet der Return Response auf das versenden der E-Mail und könnte auch Wegbrechen wenn der E-Mail Versand fehl schlägt...

Reply:
guter punkt. dann werd ich das auch bei dem anderen notification mailer so mitmachen, oder?

Reply:
sicher, dass `deliver_later` auch geht? nutzen wir bisher noch nirgends in der mainserver anwendung.

Reply:
> [ActiveJob] Enqueued ActionMailer::DeliveryJob (Job ID: 12a76437-78b4-4ed4-8578-ae3d381e85a8) to DelayedJob(mailers) with arguments: "NotificationMailer", "survey_commented", "deliver_now", #<GlobalID:0x00007fd0e91f3468 @uri=#<URI::GID gid://smart-village-app-mainserver/Survey::Comment/18>>, #<GlobalID:0x00007fd0e91f2d38 @uri=#<URI::GID gid://smart-village-app-mainserver/Survey::Poll/21>>

ein `deliver_later` scheint es erfolgreich in die queue zu packen. aber bin mir unsicher, ob es `delayed_job`s gibt zum abarbeiten. ein `be delayed_job start` sagt:

> bundler: command not found: delayed_job
> Install missing gem executables with `bundle install`

Reply:
`bin/delayed_job start` is der trick!

Reply:
done in 5bdb43a